DSharpPlus/Entities/Channel/DiscordChannel.cs

```diff
@@ -212,6 +212,12 @@ public virtual IReadOnlyList<DiscordMember> Users
         public DiscordVoiceRegion RtcRegion
             => this.RtcRegionId != null ? this.Discord.VoiceRegions[this.RtcRegionId] : null;
 
+        /// <summary>
+        /// Only sent on the resolved channels of interaction responses for application commands. Gets the permissions of the user in this channel who invoked the command.
```

Comment:
The first sentence should be a `<para>`, with the second sentence being the first thing the user reads.